code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -294,3 +294,73 @@
 			host.embedded -= src
 			host.drop_from_inventory(src)
 		spawn(1) if(src) qdel(src)
+
+/*
+ *Energy Spear
+ */
+
+ /obj/item/weapon/melee/energy/spear
+ 	name = "energy spear"
+ 	desc = "Concentrated energy forming a sharp tip at the end of a long rod."
+ 	icon_state = "espear0"
+ 	armor_penetration = 75
+ 	sharp = 1
+ 	edge = 1
+ 	force = 5
+ 	throwforce = 10
+ 	throw_speed = 7
+ 	throw_range = 11
+ 	reach = 2
+ 	w_class = ITEMSIZE_LARGE
+ 	active_force = 25
+ 	active_throwforce = 40

```

Comment:
Why is this 40?

Reply:
I don't do the balance, I just put down the numbers. What would you suggest? Just copy-paste stats from the regular spear?

Reply:
30 for active_throwforce, maybe?